app/styles/components/_recently-updated-display.scss

```diff
@@ -0,0 +1,5 @@
+.recently-updated-display {
```

Comment:
@djvoa12 this is duplicate style from ilios-calendar - we shouldn't need it in both places.